tests/ffi/alias.rs

```diff
@@ -1,13 +1,20 @@
 // Rustfmt mangles the extern type alias.
 // https://github.com/rust-lang/rustfmt/issues/4159
 #[rustfmt::skip]
-#[cxx::bridge(namespace = tests)]
+#[cxx::bridge(namespace = alias_tests)]
 pub mod ffi {
     extern "C" {
         include!("cxx-test-suite/tests.h");
 
+        // Review TODO: Unquoted namespace here doesn't work, is that expected or a bug
+        // in my parsing?
+        #[namespace = "tests"]
```

Comment:
I'm new to proc-macros and usage of syn so it's unclear to me whether an unquoted namespace here should be able to work, as it does above.

```
error: unexpected token: `tests`
  --> tests/ffi/alias.rs:10:23
   |
6  |     extern "C" {
   |                - while parsing this item list starting here
...
10 |         #[namespace = tests]
   |                       ^^^^^
...
18 |     }
   |     - the item list ends here
```

Reply:
Obsolete